cli/wallet/validator.go

```diff
@@ -122,10 +119,11 @@ func handleCandidate(ctx *cli.Context, method string, sysGas int64) error {
 	if err != nil {
 		return err
 	}
-	w := io.NewBufBinWriter()
-	emit.AppCall(w.BinWriter, neoContractHash, method, callflag.States, acc.PrivateKey().PublicKey().Bytes())
-	emit.Opcodes(w.BinWriter, opcode.ASSERT)
-	res, err := c.SignAndPushInvocationTx(w.Bytes(), acc, sysGas, gas, []rpcclient.SignerAccount{{
+	script, err := smartcontract.CreateCallWithAssertScript(neoContractHash, method, acc.PrivateKey().PublicKey().Bytes())
+	if err != nil {
+		return err
```

Comment:
`cli.NewExitError`?

Reply:
Yeah, I've noticed this also, but for some reason thought that it's an internal function the result of which is wrapped outside if needed. But that's not the case, so `NewExitError` is appropriate here. Fixed.